string_forloop.py

```diff
@@ -0,0 +1,14 @@
+#Enter a String
+a = input("Enter a String: ")
+print('your first string value is {0}'.format(a))
+print(f"Print characters of {a} ")
+
+
+#print the characters in your string
+for i in range(len(a)):
+    print(a[i])
```

Comment:
This will not work. "i" here is the value at the index itself, ie the character from the string. Also, add the reversing logic as well.